ee/agent/flags/flag_controller.go

```diff
@@ -420,6 +420,14 @@ func (fc *FlagController) OsqueryTlsDistributedWriteEndpoint() string {
 	return fc.cmdLineOpts.OsqueryTlsDistributedWriteEndpoint
 }
 
+func (fc *FlagController) OsqueryVersion() string {
```

Comment:
This should probably match the `RunningOsqueryVersion` name

Reply:
removed all the logic from runner and updated the version from the LatestOsquerydPath in knapsack